plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/data/managers/ContentValueManager.java

```diff
@@ -95,7 +99,20 @@ public void run() {
                             DBWorkbench.getPlatformUI().showError("Data is empty", "Can not save null data value");
                         }
                         if (value instanceof DBDContent) {
-                            getDBDContent(value);
+                            boolean isExternalFileOpened = false;
+                            IStreamValueEditor<Control> streamEditor
+                                = ((ContentPanelEditor) activeEditor).getStreamEditor();
+                            if (streamEditor instanceof IStreamValueEditorPersistent) {
+                                Path externalFilePath =
+                                    ((IStreamValueEditorPersistent) streamEditor).getExternalFilePath(activeEditor.getControl());
+                                File externalFile = externalFilePath == null ? null : externalFilePath.toFile();
```

Comment:
What's the point of converting back to `File`? It introduces redundant null check here